src/v2/spec.yaml

```diff
@@ -3912,7 +3995,7 @@ paths:
     get:
       tags:
         - User
-      summary: Search for Usernames
+      summary: Usernames
```

Comment:
`Search usernames` - "search" is an important part of the API here. There is a different API to fetch by a direct username that does not return search results